code/game/turfs/simulated/water.dm

```diff
@@ -0,0 +1,13 @@
+/turf/open/water
```

Comment:
Acidic(?) jungle water should probably function differently than shallow pool/beach water anyway